widgets/pulseaudio.lua

```diff
@@ -31,7 +31,15 @@ local function factory(args)
     --  '/volume:/ p' prints the line containing 'volume:'.
     --  '/muted:/ p' prints the line containing 'muted:'.
     --  '/device\\.string/ p' prints the line containing 'device.string:'.
-
+    
+    local settings    = args.settings or function()
```

Comment:
Like `worker`: either add the same functionality to all widgets, or remove it.

I don't agree on this anyway: we can't assume a default settings, nor is that difficoult to write a `settings` function.